### ISSUE TITLE
Install Keycloak Operator in CLAB

### ISSUE THREAD

**Issue by lukegonis** [BCDevOps/developer-experience#4747]
**Describe the issue**
- We received a request from Zorin (PO of SSO Team) to install the Red Hat Keycloak Operator in CLAB. This is needed because Keycloak is a critical service used by many apps on our platforms.

**How does this benefit the users of our platform?**

**Definition of done**
- [x] Operator installed in CLAB
- [x] Overview provided to SSO Team
- [ ] Elevated privileges granted in CCM Repo

**Comment by StevenBarre** [comment]
Installed the operator, but found it only supports single namespace installation mode, and not a global mode. The operator also does not officially support having multiple copies installed in a cluster as would be needed for proper dev/test/prod setup across namespaces.

In a subsequent call with Red Hat we found that the operator is not a requirement and existing deployments can be updated to the new image from RH without using the operator.

Operator was then uninstalled from CLAB and the CRDs were removed.
